libs/solid/skeleton/src/lib/components/landing/landing.component.ts

```diff
@@ -37,18 +38,30 @@ export const SOLID_SKELETON_HACKY_INJECTION = new InjectionToken<() => void>(
 })
 export class LandingComponent implements OnInit, AfterViewInit, OnDestroy {
   @Select(MenuState.getLandingItems)
-  public MenuItems!: Observable<MenuItem[]>;
+  public menuItems$!: Observable<MenuItem[]>;
 
   @ViewChild('landing') Landing?: ElementRef;
   public onGlossaryClick = new EventEmitter();
 
+  private landingBannerKey = 'hide_landing_banner';
```

Comment:
Ist das mit den Keys ein Refactoring?

Reply:
Yup. Wir brauchen diese Keys mehrmals im Code, deswegen habe ich die in einem Property gespeichert